ring/partition_ring_http_test.go

```diff
@@ -47,4 +55,30 @@ func TestPartitionRingPageHandler(t *testing.T) {
 
 	assert.Equal(t, http.StatusOK, recorder.Code)
 	assert.Equal(t, "text/html", recorder.Header().Get("Content-Type"))
+
+	doc, err := goquery.NewDocumentFromReader(recorder.Body)
+	require.NoError(t, err)
+
+	rows := doc.Find("table tbody tr")
+	require.Equal(t, 3, rows.Length())
+
+	row := doc.Find("table tbody tr:nth-child(1)")
+	require.Equal(t, 1, row.Length())
+	assert.Equal(t, "1", strings.TrimSpace(row.Find("td:nth-child(1)").Text()))
+	assert.Equal(t, "Active", strings.TrimSpace(row.Find("td:nth-child(2)").Text()))
+	assert.Contains(t, strings.TrimSpace(row.Find("td:nth-child(4)").Text()), "ingester-zone-a-0")
+	assert.Contains(t, strings.TrimSpace(row.Find("td:nth-child(4)").Text()), "ingester-zone-b-0")
```

Comment:
do we need `TrimSpace` if we're already using `Contains`? Is it possible to use an exact match - in that case it's less likely we mess up the query and match `Contains` on a big chunk of the HTML page

Reply:
I've removed goquery dependency at all and used regexps so we don't introduce a big dependecy just because of this test.